frontend/src/lib/k8s/pod.ts

```diff
@@ -65,8 +66,8 @@ class Pod extends makeKubeObject<KubePod>('Pod') {
     return cancel;
   }
 
-  exec(container: string, onExec: StreamResultsCb, options: ExecOptions = {}) {
-    const { command = ['sh'], reconnectOnFailure = undefined } = options;
+  exec(container: string, shell: string, onExec: StreamResultsCb, options: ExecOptions = {}) {
+    const { command = [shell], reconnectOnFailure = undefined } = options;
```

Comment:
I don't think we need to add `shell` as an argument, when it can be taken from ExecOptions, which is more future-proof.